.github/workflows/build.yaml

```
@@ -34,6 +48,13 @@ jobs:
         ref: ${{ steps.kernel-submodule.outputs.ref }}
         path: agnos-kernel-sdm845
 
+    - name: ccache
+      uses: hendrikmuhs/ccache-action@v1.2
```

external actions should always be pinned to a commit
```suggestion
      uses: hendrikmuhs/ccache-action@c92f40bee50034e84c763e33b317c77adaa81c92
```